components/TopBar.tsx

```diff
@@ -0,0 +1,40 @@
+import { AppBar, Box, Button, Toolbar, Typography, useTheme } from '@mui/material';
+import React, { useContext } from 'react';
+import { ColorModeToggleSwitch } from './ColorModeToggleSwitch';
+import { ColorModeContext } from '../utils/ColorModeContext';
+
+// declare react component for top bar
+const TopBar: React.FC = () => {
+  const colorMode = useContext(ColorModeContext);
+  const theme = useTheme();
+  return (
+    <AppBar position="static">
+      <Toolbar>
+        <Box sx={{ flexGrow: 1 }}>
+          <Typography
+            variant="h6"
+            color="inherit"
+            component="div"
+            display="inline"
+            mr={2}
```

Comment:
Added a margin to the right of the title to separate it a little from buttons

Reply:
Cool